torchvision/transforms/transforms.py

```diff
@@ -1837,6 +1838,80 @@ def __repr__(self) -> str:
         return s
 
 
+class GaussianNoise(torch.nn.Module):
+    """Adds Gaussian noise to the image with specified mean and standard deviation.
+    If the image is torch Tensor, it is expected
+    to have [..., C, H, W] shape, where ... means an arbitrary number of leading dimensions.
+
+    Args:
+        mean (float or sequence): Mean of the sampling gaussian distribution .
+        sigma (float or tuple of float (min, max)): Standard deviation to be used for
+            sampling the gaussian noise. If float, sigma is fixed. If it is tuple
+            of float (min, max), sigma is chosen uniformly at random to lie in the
+            given range.
+
+    Returns:
+        PIL Image or Tensor: Gaussian blurred version of the input image.
+
+    """
+
+    def __init__(self, mean, sigma=(0.1, 2.0)):
```

Comment:
```suggestion
    def __init__(self, mean: float, sigma: Optional[float, Tuple[float, float]] = (0.1, 2.0)):
```

Not sure if Tuple[float, float] goes well with with JIT. Might need to check.

Reply:
Aah yes my bad

Reply:
@parth-shastri please address this comment